File: 0283-move-zeroes/0283-move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int insert = 0;
        for(int i=0; i < nums.length; i++){
            if(nums[i] != 0){
                nums[insert] = nums[i];
                insert ++;
            }
        }
        for(int i = insert; i < nums.length; i++){
            nums[i]=0;
        }
    }
}

File: 0283-move-zeroes/0283-move-zeroes.py
class Solution(object):
    def moveZeroes(self, nums):
        insert = 0
        for i in range(len(nums)):
            if nums[i] !=0:
                nums[insert] = nums[i]
                insert += 1
        for i in range(insert, len(nums)):
            nums[i] = 0
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
        